lib/streamlit/elements/vega_charts.py

```diff
@@ -402,10 +503,11 @@ def area_chart(
             width=width,
             height=height,
         )
-        marshall(proto, chart, use_container_width, theme="streamlit")
-
-        return self.dg._enqueue(
-            "arrow_area_chart", proto, add_rows_metadata=add_rows_metadata
+        return self._altair_chart(
```

Comment:
nit: it might be a good idea to leave a small comment about how Vega and Altair belong together, because the class is called `VegaChartsMixin` but then here we return an `altair_chart`. This could be confusing to people who do not know about these projects

Reply:
Added a docstring comment to the VegaChartsMixin class to explain how the commands are related 👍